api/README.md

```diff
@@ -101,3 +113,24 @@ Alternatively, if you don't want to use your ~/.zuliprc file:
         --api-key a0b1c2d3e4f5a6b7c8d9e0f1a2b3c4d5 \
         hamlet@example.com cordelia@example.com -m \
         "Conscience doth make cowards of us all."
+
+#### Working with an untrusted server certificate
+
+If your server has either a self-signed certificate, or a certificate signed
+by a CA that you don't wish to globally trust then by default the API will
+fail with an SSL verification error.
+
+There are two options:
+
+    Add "insecure=true" to your .zuliprc file.
+    This disables verification of the server certificate, so connections are
+    encrypted but unauthenticated. This is not secure, but may be good enough
+    for a development environment.
+
+    Save the server certificate (or the CA certificate) in its own file,
+    converting to PEM format first if necessary.
+    Verify that the certificate you have saved is the same as the one on the
+    server.
+    Add "cert_bundle=<filename>" to your .zuliprc file.
+    This option trusts the server / CA certificate only for interaction with
+    the zulip site, and is relatively secure.
```

Comment:
I'm not really happy with it either, but markdown does not allow quoted blocks within a list.
I think the version I crossposted with this comment is better formatted.